### DISCUSSION TITLE
Index with two collections

### DISCUSSION THREAD

**Discussion by MohMehrnia** [ravendb/ravendb#19383] [General]
How do create index from two collection,
I must join two collection with condition then create reduce

**Comment by karmeli87** [comment]
Hi,

To index multiple collections you can use [multi-map index](https://ravendb.net/docs/article-page/6.2/csharp/indexes/map-reduce-indexes#creating-multi-map-reduce-indexes)

You cannot join, but instead you can load a document from a different collection with the use of `LoadDocument`, check this [example](https://ravendb.net/docs/article-page/6.2/Csharp/indexes/map-reduce-indexes#creating-map-reduce-indexes) of how to achieve this.
